docs/DatasetEvaluation.md

```diff
@@ -0,0 +1,263 @@
+# Dataset Evaluation
+
+This guide shows you how to evaluate the value/quality of a synthetic dataset by training a [Faster-RCNN](https://arxiv.org/abs/1506.01497) object detection model on it and testing the performance of that model on a well-known held out dataset of real images
+
```

Comment:
- We can have list of topics in the beginning of this page.
- We may want to take common steps out of every pipeline testing process and refer there as needed.

Reply:
The common steps are repeated on-purpose to make sure readers does not have to jump between sections which breaks the flow of following this instructions. Users are not expected to go through every sections of this document. Instead, they are encouraged to skip "(optional)" sections.

What do you mean by list of topics in the beginning of this page? Do you mean something like table of content that links to each section?